convert2rhel/actions/system_checks/package_updates.py

```diff
@@ -79,48 +77,23 @@ def run(self):
             return
 
         if len(packages_to_update) > 0:
-            package_not_up_to_date_skip = os.environ.get("CONVERT2RHEL_OUTDATED_PACKAGE_CHECK_SKIP", None)
             package_not_up_to_date_error_message = (
                 "The system has %s package(s) not updated based on repositories defined in the system repositories.\n"
                 "List of packages to update: %s.\n\n"
                 "Not updating the packages may cause the conversion to fail.\n"
                 "Consider updating the packages before proceeding with the conversion."
                 % (len(packages_to_update), " ".join(packages_to_update))
             )
-            if not package_not_up_to_date_skip:
-                logger.warning(package_not_up_to_date_error_message)
-                self.set_result(
-                    level="OVERRIDABLE",
-                    id="OUT_OF_DATE_PACKAGES",
-                    title="Outdated packages detected",
-                    description="Please refer to the diagnosis for further information",
-                    diagnosis=package_not_up_to_date_error_message,
-                    remediations="If you wish to ignore this message, set the environment variable "
-                    "'CONVERT2RHEL_OUTDATED_PACKAGE_CHECK_SKIP' to 1.",
-                )
-                return
-
-            skip_package_not_up_to_date_message = (
-                "Detected 'CONVERT2RHEL_OUTDATED_PACKAGE_CHECK_SKIP' environment variable, we will skip "
-                "the package up-to-date check.\n"
-                "Beware, this could leave your system in a broken state."
-            )
-            logger.warning(skip_package_not_up_to_date_message)
-            self.add_message(
-                level="WARNING",
-                id="SKIP_OUTDATED_PACKAGE_CHECK",
-                title="Skip package not up to date check",
-                description=skip_package_not_up_to_date_message,
-            )
-
             logger.warning(package_not_up_to_date_error_message)
             self.add_message(
                 level="WARNING",
-                id="OUTDATED_PACKAGE_MESSAGE",
+                id="OUT_OF_DATE_PACKAGES",
                 title="Outdated packages detected",
```

Comment:
This message was output if the `CONVERT2RHEL_OUTDATED_PACKAGE_CHECK_SKIP` env var was used to skip the OVERRIDABLE inhibitor that was in place. Since we have changed the OVERRIDABLE inhibitor to a warning now, we no longer need `OUTDATED_PACKAGE_MESSAGE`. `OUT_OF_DATE_PACKAGES` isn't a new ID, it is just formatted liked this because the lines from `OUTDATED_PACKAGE_MESSAGE` match what was there for the `OUT_OF_DATE_PACKAGES` result. This is how the final formatting looks - https://github.com/oamg/convert2rhel/blob/b8d3c5734561bf0ea4333215d516db2034b4752c/convert2rhel/actions/system_checks/package_updates.py#L9